pytorch_lightning/trainer/properties.py

```diff
@@ -223,8 +224,13 @@ def model(self, model: torch.nn.Module) -> None:
     def log_dir(self) -> Optional[str]:
         if self.logger is None:
             dirpath = self.default_root_dir
+        elif isinstance(self.logger, TensorBoardLogger):
+            dirpath = self.logger.log_dir
```

Comment:
this is inconsistent, then let's add `save_dir` property to `TensorBoardLogger`

Reply:
If i'm not missing something, the current changes keep the same functionality.

Adding `save_dir` would be for a separate PR IMO